### ISSUE TITLE
Previously Installed Jupyter distribution cant find Julia

### ISSUE THREAD

**Issue by Haru-Fujimura** [JuliaLang/IJulia.jl#802]
So I have a previously installed version of `conda` and `jupyter` that I want to use. Since the `conda` distribution for IJulia is currently not working on windows, I attempted to simply install Julia, change the ENV variable to

```
ENV["JUPYTER"] = "C:\\Users\\Haru\\Miniconda3\\share\\jupyter\\" 
```
then installed IJulia the normal way:

```
using Pkg
Pkg.add("IJulia")
```
and then i opened my previous Jupyter distribution and when i attempted to start the Julia Kernel i recieved the error:
<details>

```
Traceback (most recent call last):
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\web.py", line 1592, in _execute
    result = yield result
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\notebook\services\sessions\handlers.py", line 73, in post
    type=mtype))
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\notebook\services\sessions\sessionmanager.py", line 79, in create_session
    kernel_id = yield self.start_kernel_for_session(session_id, path, name, type, kernel_name)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\notebook\services\sessions\sessionmanager.py", line 92, in start_kernel_for_session
    self.kernel_manager.start_kernel(path=kernel_path, kernel_name=kernel_name)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\Users\Haru\Miniconda3\lib\site-packages\tornado\gen.py", line 326, in wrapper
    yielded = next(result)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\notebook\services\kernels\kernelmanager.py", line 160, in start_kernel
    super(MappingKernelManager, self).start_kernel(**kwargs)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\jupyter_client\multikernelmanager.py", line 110, in start_kernel
    km.start_kernel(**kwargs)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\jupyter_client\manager.py", line 259, in start_kernel
    **kw)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\jupyter_client\manager.py", line 204, in _launch_kernel
    return launch_kernel(kernel_cmd, **kw)
  File "C:\Users\Haru\Miniconda3\lib\site-packages\jupyter_client\launcher.py", line 138, in launch_kernel
    proc = Popen(cmd, **kwargs)
  File "C:\Users\Haru\Miniconda3\lib\subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "C:\Users\Haru\Miniconda3\lib\subprocess.py", line 997, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified


```
</details> 

Is this because of Jupyter not knowing  where Julia is installed? How do I tell it where for it to look for Julia?

**Comment by stevengj** [comment]
You don't need to set `JUPYTER` — the kernel configuration file is installed in a location that should be visible to all Jupyter installations.   Have you tried `Pkg.build("IJulia")` in case your Julia installation has moved since you first installed it?

**Comment by navneethc** [comment]
I hope @Haru-Fujimura doesn't mind me slightly diverting this thread, since I, as a new Julia user, didn't want to open a new issue on generally the same topic. :) (Please let me know if you wish to shift this to a separate issue.)

I have the official Windows version 1.10 running, IJulia is installed, ENV["JUPYTER"] pointing to an Anaconda-installed Jupyter executable, and, unlike Haru, I can run cells in the notebook if I open Jupyter first and select the Julia kernel. However, when I try to run Jupyter from the Julia shell, I get the following errors. Are these expected? Is there some other way of directly opening a notebook with the Julia kernel running?
```

julia> ENV["JUPYTER"]="C:\\Users\\Navneeth\\Miniconda3\\Scripts\\jupyter"
"C:\\Users\\Navneeth\\Miniconda3\\Scripts\\jupyter"

julia> using Pkg

julia> Pkg.add("IJulia")
  Updating registry at `C:\Users\Navneeth\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
  Updating `C:\Users\Navneeth\.julia\environments\v1.1\Project.toml`
  [7073ff75] + IJulia v1.17.0
  Updating `C:\Users\Navneeth\.julia\environments\v1.1\Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.3
  [34da2185] + Compat v1.5.1
  [8f4d0f93] + Conda v1.2.0
  [7073ff75] + IJulia v1.17.0
  [682c06a0] + JSON v0.20.0
  [739be429] + MbedTLS v0.6.7
  [b85f4697] + SoftGlobalScope v1.0.9
  [81def892] + VersionParsing v1.1.3
  [c2297ded] + ZMQ v1.0.0
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [8bb1440f] + DelimitedFiles
  [8ba89e20] + Distributed
  [7b1f6079] + FileWatching
  [b77e0a4c] + InteractiveUtils
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [1a1011a3] + SharedArrays
  [6462fe0b] + Sockets
  [2f01184e] + SparseArrays
  [10745b16] + Statistics
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode

julia> using IJulia

julia> notebook()
install Jupyter via Conda, y/n? [y]: n
ERROR: C:\Users\Navneeth\.julia\conda\3\Scripts\jupyter.exe is not installed, cannot run notebook
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] find_jupyter_subcommand(::String) at C:\Users\Navneeth\.julia\packages\IJulia\4UizY\src\jupyter.jl:25
 [3] #notebook#8 at C:\Users\Navneeth\.julia\packages\IJulia\4UizY\src\jupyter.jl:88 [inlined]
 [4] notebook() at C:\Users\Navneeth\.julia\packages\IJulia\4UizY\src\jupyter.jl:87
 [5] top-level scope at none:0

julia> jupyter notebook
ERROR: syntax: extra token "notebook" after end of expression

julia> jupyter
ERROR: UndefVarError: jupyter not defined

julia>
```

**Comment by Haru-Fujimura** [comment]
Im pretty sure thats fine @navneethc since this is probably a general problem with the environment variable anyway.

**Comment by stevengj** [comment]
does `C:\Users\Navneeth\.julia\conda\3\Scripts\jupyter.exe` exist?

**Comment by navneethc** [comment]
> 
> 
> does `C:\Users\Navneeth\.julia\conda\3\Scripts\jupyter.exe` exist?

There is not `Scripts` directory under `3`.

```
C:\Users\Navneeth\.julia\conda\3
λ ls -a
./  ../
```

**Comment by stevengj** [comment]
Sounds like there was an error with the Conda installation.  What happens if you do
```jl
using Conda
Conda.add("jupyter")
```

**Comment by navneethc** [comment]
```

julia> using Conda
ERROR: ArgumentError: Package Conda not found in current path:
- Run `import Pkg; Pkg.add("Conda")` to install the Conda package.

Stacktrace:
 [1] require(::Module, ::Symbol) at .\loading.jl:823

julia>
```

Just to clarify, the conda I have is from Miniconda (originally installed for Python stuff). I have not installed Conda via Julia.

**Comment by stevengj** [comment]
@navneethc, you *have* installed Conda via Julia — that is why the error message above mentions `C:\Users\Navneeth\.julia\conda\3`, which is the Julia-internal Conda installation.  (This is the default on Windows.)

The Conda package was installed only as a dependency of IJulia, so it is not user-visible (`using Conda` gives an error) even though it is internally present.  You have to do `add Conda` first to make it user-visible.

**Comment by navneethc** [comment]
My bad, still getting used to things in this language. 

`add Conda` gives me the following error: 
```
julia> add Conda
ERROR: syntax: extra token "Conda" after end of expression
```

So, I googled to find out how to "add" a package. I tried something and I'm afraid that I ended up with completely redundant copies of Miniconda, Jupyter and its dependencies, which is exactly what I didn't want. :(

WARNING: long console output below.
<details>
<summary>Click to expand</summary>

```
julia> using Pkg

julia> Pkg.add("Conda")
  Updating registry at `C:\Users\Navneeth\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed Compat ─────── v2.0.0
 Installed StatsBase ──── v0.29.0
 Installed MbedTLS ────── v0.6.8
 Installed StaticArrays ─ v0.10.3
  Updating `C:\Users\Navneeth\.julia\environments\v1.1\Project.toml`
  [8f4d0f93] + Conda v1.2.0
  Updating `C:\Users\Navneeth\.julia\environments\v1.1\Manifest.toml`
  [34da2185] ↑ Compat v1.5.1 ⇒ v2.0.0
  [739be429] ↑ MbedTLS v0.6.7 ⇒ v0.6.8
  [90137ffa] ↑ StaticArrays v0.10.2 ⇒ v0.10.3
  [2913bbd2] ↑ StatsBase v0.27.0 ⇒ v0.29.0
  Building MbedTLS → `C:\Users\Navneeth\.julia\packages\MbedTLS\X4xar\deps\build.log`

julia> using Conda
[ Info: Recompiling stale cache file C:\Users\Navneeth\.julia\compiled\v1.1\Conda\WZE3U.ji for Conda [8f4d0f93-b110-5947-807f-2305c1781a2d]

julia> Conda.add("jupyter")
[ Info: Downloading miniconda installer ...
[ Info: Installing miniconda ...
[ Info: Running `conda config --add channels defaults --file 'C:\Users\Navneeth\.julia\conda\3\condarc-julia.yml' --force` in root environment
Warning: 'defaults' already in 'channels' list, moving to the top
[ Info: Running `conda update -y conda` in root environment
Solving environment: done

## Package Plan ##

  environment location: C:\Users\Navneeth\.julia\conda\3

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    certifi-2018.11.29         |           py36_0         146 KB
    vs2015_runtime-14.15.26706 |       h3a45250_0         2.2 MB
    ca-certificates-2019.1.23  |                0         158 KB
    cryptography-2.5           |   py36h7a1dbc1_0         573 KB
    vc-14.1                    |       h0510ff6_4           6 KB
    openssl-1.1.1b             |       he774522_0         5.8 MB
    conda-4.6.7                |           py36_0         1.7 MB
    ------------------------------------------------------------
                                           Total:        10.5 MB

The following packages will be UPDATED:

    ca-certificates: 2018.03.07-0         --> 2019.1.23-0
    certifi:         2018.4.16-py36_0     --> 2018.11.29-py36_0
    conda:           4.5.4-py36_0         --> 4.6.7-py36_0
    cryptography:    2.2.2-py36hfa6e2cd_0 --> 2.5-py36h7a1dbc1_0
    openssl:         1.0.2o-h8ea7d77_0    --> 1.1.1b-he774522_0
    vc:              14-h0510ff6_3        --> 14.1-h0510ff6_4
    vs2015_runtime:  14.0.25123-3         --> 14.15.26706-h3a45250_0


Downloading and Extracting Packages
certifi-2018.11.29   |  146 KB | #################################################################################################################### | 100%
vs2015_runtime-14.15 |  2.2 MB | #################################################################################################################### | 100%
ca-certificates-2019 |  158 KB | #################################################################################################################### | 100%
cryptography-2.5     |  573 KB | #################################################################################################################### | 100%
vc-14.1              |    6 KB | #################################################################################################################### | 100%
openssl-1.1.1b       |  5.8 MB | #################################################################################################################### | 100%
conda-4.6.7          |  1.7 MB | #################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
[ Info: Running `conda install -y jupyter` in root environment
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\Navneeth\.julia\conda\3

  added / updated specs:
    - jupyter


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    backcall-0.1.0             |           py36_0          20 KB
    bleach-3.1.0               |           py36_0         224 KB
    colorama-0.4.1             |           py36_0          24 KB
    decorator-4.3.2            |           py36_0          17 KB
    entrypoints-0.3            |           py36_0          12 KB
    icu-58.2                   |       ha66f8fd_1        21.9 MB
    ipykernel-5.1.0            |   py36h39e3cac_0         156 KB
    ipython-7.3.0              |   py36h39e3cac_0         1.1 MB
    ipython_genutils-0.2.0     |   py36h3c5d0ee_0          39 KB
    ipywidgets-7.4.2           |           py36_0         151 KB
    jedi-0.13.3                |           py36_0         236 KB
    jinja2-2.10                |           py36_0         183 KB
    jpeg-9b                    |       hb83a4c4_2         313 KB
    jsonschema-2.6.0           |   py36h7636477_0         104 KB
    jupyter-1.0.0              |           py36_7           6 KB
    jupyter_client-5.2.4       |           py36_0         203 KB
    jupyter_console-6.0.0      |           py36_0          53 KB
    jupyter_core-4.4.0         |           py36_0          85 KB
    libpng-1.6.36              |       h2a8f88b_0         550 KB
    libsodium-1.0.16           |       h9d3ae62_0         585 KB
    m2w64-gcc-libgfortran-5.3.0|                6         340 KB
    m2w64-gcc-libs-5.3.0       |                7         518 KB
    m2w64-gcc-libs-core-5.3.0  |                7         213 KB
    m2w64-gmp-6.1.0            |                2         689 KB
    m2w64-libwinpthread-git-5.0.0.4634.697f757|                2          30 KB
    markupsafe-1.1.1           |   py36he774522_0          32 KB
    mistune-0.8.4              |   py36he774522_0          54 KB
    msys2-conda-epoch-20160418 |                1           2 KB
    nbconvert-5.3.1            |           py36_0         422 KB
    nbformat-4.4.0             |   py36h3a5bc1b_0         157 KB
    notebook-5.7.4             |           py36_0         7.3 MB
    pandoc-2.2.3.2             |                0        21.0 MB
    pandocfilters-1.4.2        |           py36_1          13 KB
    parso-0.3.4                |           py36_0         121 KB
    pickleshare-0.7.5          |           py36_0          13 KB
    prometheus_client-0.6.0    |           py36_0          70 KB
    prompt_toolkit-2.0.9       |           py36_0         493 KB
    pygments-2.3.1             |           py36_0         1.4 MB
    pyqt-5.9.2                 |   py36h6538335_2         4.2 MB
    python-dateutil-2.8.0      |           py36_0         282 KB
    pywinpty-0.5.5             |        py36_1000          47 KB
    pyzmq-18.0.0               |   py36ha925a31_0         465 KB
    qt-5.9.7                   |   vc14h73c81de_0        92.3 MB
    qtconsole-4.4.3            |           py36_0         176 KB
    send2trash-1.5.0           |           py36_0          16 KB
    sip-4.19.8                 |   py36h6538335_0         282 KB
    sqlite-3.26.0              |       he774522_0         936 KB
    terminado-0.8.1            |           py36_1          21 KB
    testpath-0.4.2             |           py36_0          92 KB
    tornado-5.1.1              |   py36hfa6e2cd_0         666 KB
    traitlets-4.3.2            |   py36h096827d_0         131 KB
    wcwidth-0.1.7              |   py36h3d5aa90_0          25 KB
    webencodings-0.5.1         |           py36_1          19 KB
    widgetsnbextension-3.4.2   |           py36_0         1.7 MB
    winpty-0.4.3               |                4         1.1 MB
    zeromq-4.3.1               |       h33f27b4_3        52.4 MB
    zlib-1.2.11                |       h62dcd97_3         128 KB
    ------------------------------------------------------------
                                           Total:       213.7 MB

The following NEW packages will be INSTALLED:

  backcall           pkgs/main/win-64::backcall-0.1.0-py36_0
  bleach             pkgs/main/win-64::bleach-3.1.0-py36_0
  colorama           pkgs/main/win-64::colorama-0.4.1-py36_0
  decorator          pkgs/main/win-64::decorator-4.3.2-py36_0
  entrypoints        pkgs/main/win-64::entrypoints-0.3-py36_0
  icu                pkgs/main/win-64::icu-58.2-ha66f8fd_1
  ipykernel          pkgs/main/win-64::ipykernel-5.1.0-py36h39e3cac_0
  ipython            pkgs/main/win-64::ipython-7.3.0-py36h39e3cac_0
  ipython_genutils   pkgs/main/win-64::ipython_genutils-0.2.0-py36h3c5d0ee_0
  ipywidgets         pkgs/main/win-64::ipywidgets-7.4.2-py36_0
  jedi               pkgs/main/win-64::jedi-0.13.3-py36_0
  jinja2             pkgs/main/win-64::jinja2-2.10-py36_0
  jpeg               pkgs/main/win-64::jpeg-9b-hb83a4c4_2
  jsonschema         pkgs/main/win-64::jsonschema-2.6.0-py36h7636477_0
  jupyter            pkgs/main/win-64::jupyter-1.0.0-py36_7
  jupyter_client     pkgs/main/win-64::jupyter_client-5.2.4-py36_0
  jupyter_console    pkgs/main/win-64::jupyter_console-6.0.0-py36_0
  jupyter_core       pkgs/main/win-64::jupyter_core-4.4.0-py36_0
  libpng             pkgs/main/win-64::libpng-1.6.36-h2a8f88b_0
  libsodium          pkgs/main/win-64::libsodium-1.0.16-h9d3ae62_0
  m2w64-gcc-libgfor~ pkgs/msys2/win-64::m2w64-gcc-libgfortran-5.3.0-6
  m2w64-gcc-libs     pkgs/msys2/win-64::m2w64-gcc-libs-5.3.0-7
  m2w64-gcc-libs-co~ pkgs/msys2/win-64::m2w64-gcc-libs-core-5.3.0-7
  m2w64-gmp          pkgs/msys2/win-64::m2w64-gmp-6.1.0-2
  m2w64-libwinpthre~ pkgs/msys2/win-64::m2w64-libwinpthread-git-5.0.0.4634.697f757-2
  markupsafe         pkgs/main/win-64::markupsafe-1.1.1-py36he774522_0
  mistune            pkgs/main/win-64::mistune-0.8.4-py36he774522_0
  msys2-conda-epoch  pkgs/msys2/win-64::msys2-conda-epoch-20160418-1
  nbconvert          pkgs/main/win-64::nbconvert-5.3.1-py36_0
  nbformat           pkgs/main/win-64::nbformat-4.4.0-py36h3a5bc1b_0
  notebook           pkgs/main/win-64::notebook-5.7.4-py36_0
  pandoc             pkgs/main/win-64::pandoc-2.2.3.2-0
  pandocfilters      pkgs/main/win-64::pandocfilters-1.4.2-py36_1
  parso              pkgs/main/win-64::parso-0.3.4-py36_0
  pickleshare        pkgs/main/win-64::pickleshare-0.7.5-py36_0
  prometheus_client  pkgs/main/win-64::prometheus_client-0.6.0-py36_0
  prompt_toolkit     pkgs/main/win-64::prompt_toolkit-2.0.9-py36_0
  pygments           pkgs/main/win-64::pygments-2.3.1-py36_0
  pyqt               pkgs/main/win-64::pyqt-5.9.2-py36h6538335_2
  python-dateutil    pkgs/main/win-64::python-dateutil-2.8.0-py36_0
  pywinpty           pkgs/main/win-64::pywinpty-0.5.5-py36_1000
  pyzmq              pkgs/main/win-64::pyzmq-18.0.0-py36ha925a31_0
  qt                 pkgs/main/win-64::qt-5.9.7-vc14h73c81de_0
  qtconsole          pkgs/main/win-64::qtconsole-4.4.3-py36_0
  send2trash         pkgs/main/win-64::send2trash-1.5.0-py36_0
  sip                pkgs/main/win-64::sip-4.19.8-py36h6538335_0
  sqlite             pkgs/main/win-64::sqlite-3.26.0-he774522_0
  terminado          pkgs/main/win-64::terminado-0.8.1-py36_1
  testpath           pkgs/main/win-64::testpath-0.4.2-py36_0
  tornado            pkgs/main/win-64::tornado-5.1.1-py36hfa6e2cd_0
  traitlets          pkgs/main/win-64::traitlets-4.3.2-py36h096827d_0
  wcwidth            pkgs/main/win-64::wcwidth-0.1.7-py36h3d5aa90_0
  webencodings       pkgs/main/win-64::webencodings-0.5.1-py36_1
  widgetsnbextension pkgs/main/win-64::widgetsnbextension-3.4.2-py36_0
  winpty             pkgs/main/win-64::winpty-0.4.3-4
  zeromq             pkgs/main/win-64::zeromq-4.3.1-h33f27b4_3
  zlib               pkgs/main/win-64::zlib-1.2.11-h62dcd97_3



Downloading and Extracting Packages
colorama-0.4.1       | 24 KB     | ################################################################################################################## | 100%
pyzmq-18.0.0         | 465 KB    | ################################################################################################################## | 100%
m2w64-gmp-6.1.0      | 689 KB    | ################################################################################################################## | 100%
prometheus_client-0. | 70 KB     | ################################################################################################################## | 100%
qtconsole-4.4.3      | 176 KB    | ################################################################################################################## | 100%
sqlite-3.26.0        | 936 KB    | ################################################################################################################## | 100%
libpng-1.6.36        | 550 KB    | ################################################################################################################## | 100%
markupsafe-1.1.1     | 32 KB     | ######################################################################################################################### | 100%
winpty-0.4.3         | 1.1 MB    | ######################################################################################################################### | 100%
pandocfilters-1.4.2  | 13 KB     | ######################################################################################################################### | 100%
jupyter_core-4.4.0   | 85 KB     | ######################################################################################################################### | 100%
jupyter-1.0.0        | 6 KB      | ######################################################################################################################### | 100%
pandoc-2.2.3.2       | 21.0 MB   | ######################################################################################################################### | 100%
python-dateutil-2.8. | 282 KB    | ######################################################################################################################### | 100%
terminado-0.8.1      | 21 KB     | ######################################################################################################################### | 100%
prompt_toolkit-2.0.9 | 493 KB    | ######################################################################################################################### | 100%
jupyter_console-6.0. | 53 KB     | ######################################################################################################################### | 100%
widgetsnbextension-3 | 1.7 MB    | ######################################################################################################################### | 100%
ipywidgets-7.4.2     | 151 KB    | ######################################################################################################################### | 100%
tornado-5.1.1        | 666 KB    | ######################################################################################################################### | 100%
jedi-0.13.3          | 236 KB    | ######################################################################################################################### | 100%
jinja2-2.10          | 183 KB    | ######################################################################################################################### | 100%
pygments-2.3.1       | 1.4 MB    | ######################################################################################################################### | 100%
qt-5.9.7             | 92.3 MB   | ######################################################################################################################### | 100%
decorator-4.3.2      | 17 KB     | ######################################################################################################################### | 100%
mistune-0.8.4        | 54 KB     | ######################################################################################################################### | 100%
ipython_genutils-0.2 | 39 KB     | ######################################################################################################################### | 100%
msys2-conda-epoch-20 | 2 KB      | ######################################################################################################################### | 100%
jsonschema-2.6.0     | 104 KB    | ######################################################################################################################### | 100%
bleach-3.1.0         | 224 KB    | ######################################################################################################################### | 100%
backcall-0.1.0       | 20 KB     | ######################################################################################################################### | 100%
testpath-0.4.2       | 92 KB     | ######################################################################################################################### | 100%
pyqt-5.9.2           | 4.2 MB    | ######################################################################################################################### | 100%
parso-0.3.4          | 121 KB    | ######################################################################################################################### | 100%
sip-4.19.8           | 282 KB    | ######################################################################################################################### | 100%
send2trash-1.5.0     | 16 KB     | ######################################################################################################################### | 100%
ipykernel-5.1.0      | 156 KB    | ######################################################################################################################### | 100%
wcwidth-0.1.7        | 25 KB     | ######################################################################################################################### | 100%
icu-58.2             | 21.9 MB   | ######################################################################################################################### | 100%
ipython-7.3.0        | 1.1 MB    | ######################################################################################################################### | 100%
jpeg-9b              | 313 KB    | ######################################################################################################################### | 100%
pywinpty-0.5.5       | 47 KB     | ######################################################################################################################### | 100%
jupyter_client-5.2.4 | 203 KB    | ######################################################################################################################### | 100%
pickleshare-0.7.5    | 13 KB     | ######################################################################################################################### | 100%
nbconvert-5.3.1      | 422 KB    | ######################################################################################################################### | 100%
zlib-1.2.11          | 128 KB    | ######################################################################################################################### | 100%
entrypoints-0.3      | 12 KB     | ######################################################################################################################### | 100%
traitlets-4.3.2      | 131 KB    | ######################################################################################################################### | 100%
m2w64-gcc-libs-core- | 213 KB    | ######################################################################################################################### | 100%
notebook-5.7.4       | 7.3 MB    | ######################################################################################################################### | 100%
zeromq-4.3.1         | 52.4 MB   | ######################################################################################################################### | 100%
m2w64-gcc-libs-5.3.0 | 518 KB    | ######################################################################################################################### | 100%
m2w64-libwinpthread- | 30 KB     | ######################################################################################################################### | 100%
webencodings-0.5.1   | 19 KB     | ######################################################################################################################### | 100%
m2w64-gcc-libgfortra | 340 KB    | ######################################################################################################################### | 100%
nbformat-4.4.0       | 157 KB    | ######################################################################################################################### | 100%
libsodium-1.0.16     | 585 KB    | ######################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: - DEBUG menuinst_win32:__init__(196): Menu: name: 'Anaconda${PY_VER} ${PLATFORM}', prefix: 'C:\Users\Navneeth\.julia\conda\3', env_name: 'None', mode: 'user', used_mode: 'user'
DEBUG menuinst_win32:create(320): Shortcut cmd is C:\Users\Navneeth\.julia\conda\3\python.exe, args are ['C:\\Users\\Navneeth\\.julia\\conda\\3\\cwp.py', 'C:\\Users\\Navneeth\\.julia\\conda\\3', 'C:\\Users\\Navneeth\\.julia\\conda\\3\\python.exe', 'C:\\Users\\Navneeth\\.julia\\conda\\3\\Scripts\\jupyter-notebook-script.py', '"%USERPROFILE%/"']                                                                                                                                                    done

julia>

```
</details>

**Comment by stevengj** [comment]
You do `add Conda` at the package prompt.  At the REPL, type `]` to enter the package prompt.  See https://docs.julialang.org/en/v1/stdlib/Pkg/

**Comment by navneethc** [comment]
> 
> 
> You do `add Conda` at the package prompt. At the REPL, type `]` to enter the package prompt. See https://docs.julialang.org/en/v1/stdlib/Pkg/
> 

Ah, okay. That'll be one new thing I learn today.

> Anyway, the output indicates that it indeed installed `ipywidgets`. So, maybe the problem was that you were using an old jupyter installation from a previous Julia version or something?

No, I have installed Julia only once so far. 

By the way, is there a way not to completely remove (uninstall and delete packages) Miniconda and the stuff that it installed?

**Comment by stevengj** [comment]
> By the way, is there a way to completely remove (uninstall and delete packages) Miniconda and the stuff that it installed?
```jl
using Conda
rm(Conda.ROOTENV, recursive=true)
```

**Comment by navneethc** [comment]
Done. Thanks a lot for your help and patience with a newbie. I think I'm going to just start Jupyter and choose the kernel when needed. :)

**Comment by stevengj** [comment]
Sounds resolved now?

**Comment by hongyx11** [comment]
> You don't need to set `JUPYTER` — the kernel configuration file is installed in a location that should be visible to all Jupyter installations. Have you tried `Pkg.build("IJulia")` in case your Julia installation has moved since you first installed it?

works for me. 
Thanks
